torch_xla/csrc/runtime/metrics_reader.h

```diff
@@ -14,7 +14,7 @@ namespace metrics_reader {
 
 // Creates a report with the current metrics statistics.
 std::string CreateMetricReport(
-    const std::map<std::string, torch_xla::runtime::Metric>& xrt_metrics);
+    const std::map<std::string, torch_xla::runtime::Metric>& rt_metrics);
```

Comment:
you can just call it `metrics` instead of `rt_metrics`

Reply:
thanks for pointing, test in https://github.com/pytorch/xla/pull/5229

Reply:
Thanks, updated